examples/python-flask/README.md

```diff
@@ -0,0 +1,46 @@
+# Flask Example
+
+This project demonstrates use of verifiable credentials and presentation  for an
+application.
+
+## Dependencies
+
+- Python 3
+- Pip
+- Python 3 virtual environment
+
+```bash
+$ sudo apt update
+$ sudo apt install -y python3.6 python3-pip python3-virtualenv
+```
+
+### Python dependencies
+
+- flask-qrcode
+- Flask
+- didkit
+
+```bash
+$ python3 -m pip install flask-qrcode flask
+```
+
+### Building DIDKit
+
+DIDKit is used to handle credentials and presentations, since it's not yet
+publically available in PyPI manual installation is required.
+
+To do so got to the `lib/` folder inside this repository and run:
```

Comment:
go to*

also, aren't you specifying the Makefile dir with `-C`, so the user doesn't have to `cd` into there?

Reply:
You are correct, fixed.